tests/cases/compiler/newMap.ts

```diff
@@ -0,0 +1,2 @@
+// @lib: es6
+new Map<string>();

```

Comment:
This should’ve also had a test case for:
```ts
new WeakMap<object>();
```

---

I’m fixing that in <https://github.com/microsoft/TypeScript/pull/41635>.